.github/workflows/ami-release.yml

```diff
@@ -17,3 +17,17 @@ jobs:
       - name: Build AMI
         run: |
           packer build -var-file="development-arm.vars.pkr.hcl" -var-file="common.vars.pkr.hcl" amazon-arm64.pkr.hcl
+
+      - name: Merging migration files
+        run: cat $(ls -1) > ../migration-output.sql
+        working-directory: ${{ github.workspace }}/migrations/db/migrations
```

Comment:
Probably not needed anymore if we use migrate.sh

Reply:
discussed offline: concatenating migrations into a single file could introduce unexpected behaviour as session configs are persistent for the whole file when executed by psql, ie. `set [session | local] ...`. Will need to update `restore.ts` in a separate PR to run each migration individually.